app/assets/javascripts/discourse/app/models/bookmark.js

```diff
@@ -125,9 +125,21 @@ const Bookmark = RestModel.extend({
     ).capitalize();
   },
 
-  @discourseComputed("linked_post_number", "fancy_title", "topic_id")
-  topicLink(linked_post_number, fancy_title, id) {
-    return Topic.create({ id, fancy_title, linked_post_number });
+  @discourseComputed()
```

Comment:
Does this need to be a computed property when it is not based off any other property?